src/mixins/style-propable.js

```diff
@@ -20,14 +20,8 @@ export default {
     return ImmutabilityHelper.merge.apply(this, arguments);
   },
 
-  //Moved this function to /utils/styles.js
-  mergeAndPrefix() {
```

Comment:
I'm wondering if people aren't using this mixin in their source code and this method (I shouldn't but I do, this it's convenient :grin:).
Do you think we should add a deprecation warning?